<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AiChat</class>
 <widget class="QWidget" name="AiChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局样式表*/
QPushButton{
	border:none;
}

/*局部样式表Top*/
QWidget#Top{
	background-color: rgb(231, 234, 247);
	border-bottom:1px solid rgb(149, 149, 149);
}

QLabel#FriendHead{
	border:1px solid rgb(0, 255, 0);;
}

QPushButton#close_Btntop:hover{
	background-color: rgb(255, 0, 0);
}

QPushButton#reduce_Btn:hover{
	background-color: rgb(217, 217, 217);
}

/*局部样式表Center*/
QListWidget#listWidget{
	background-color: rgb(231, 234, 247);
}

QLabel#Image{
	border-bottom:1px solid rgb(149, 149, 149);
	background-color: rgb(255, 255, 255);
}

/*局部样式表Bottom*/
QWidget#Bottom{
    border-top:1px solid rgb(0, 0, 0);
	background-color: rgb(231, 234, 247);
}

QPushButton#emote_Btn:hover{
	 border:1px solid rgb(203, 203, 203);
}

QPushButton#file_Btn:hover{
	 border:1px solid rgb(203, 203, 203);
}

QTextEdit#text_Edit{
	background-color: rgb(231, 234, 247);
	border:noe;
}

QPushButton#close_Btnbottom{
	background-color: rgb(6, 157, 213);
	color: rgb(255, 255, 255);
	border-radius:5px;
	font:normal 18px &quot;楷体&quot;;
}

QPushButton#close_Btnbottom:hover{
	background-color: rgb(75,202,255);
}

QPushButton#sending_Btn{
	background-color: rgb(6, 157, 213);
	color: rgb(255, 255, 255);
	border-radius:5px;
	font:normal 18px &quot;楷体&quot;;
}

QPushButton#sending_Btn:hover{
	background-color: rgb(75,202,255);
}

QTextBrowser#text_Browser{
	border:none;
 	background:#E7EAF7;
}

QPushButton#setup_Btn:hover{
	background-color: rgb(217, 217, 217);
}

</string>
  </property>
  <widget class="QWidget" name="Top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>70</height>
    </size>
   </property>
   <widget class="QLabel" name="FriendHead">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FriendName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>机器人名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="FriendSign">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>机器人介绍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Btntop">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image/Image.qrc">
      <normaloff>:/new/icons/icons/becurrent/close.png</normaloff>:/new/icons/icons/becurrent/close.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="reduce_Btn">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image/Image.qrc">
      <normaloff>:/new/icons/icons/becurrent/minus.png</normaloff>:/new/icons/icons/becurrent/minus.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Centre" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>630</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>380</height>
    </size>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>630</width>
      <height>380</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>380</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>380</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Bottom" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>630</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>160</height>
    </size>
   </property>
   <widget class="QPushButton" name="close_Btnbottom">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sending_Btn">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>130</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_Edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>630</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>120</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
